pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1274,7 +1274,8 @@
         "4.1.1.0": "4.3.0",
         "4.2.0.0": "4.4.0",
         "4.4.0.0": "4.5.0",
-        "4.5.0.0": "4.6.0"
+        "4.5.0.0": "4.6.0",
+        "4.5.0.1": "4.6.1"
```

Comment:
I think you're missing assemblyVersion mapping to package version?

Reply:
I'm confused, isn't that what I'm doing here? 4.6.1 and 4.5.0.1 are the new versions as of this PR

Reply:
Oh yeah. Sorry. Misread, thought this where the stable version entry. Disregard